assets/js/common/ApiKeySettingsModal/ApiKeySettingsModal.stories.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console, react/function-component-definition */
+import React from 'react';
+import { useArgs } from '@storybook/client-api';
+import ApiKeySettingsModal from './ApiKeySettingsModal';
+
+export default {
+  title: 'Components/ApiKeySettingsModal',
+  component: ApiKeySettingsModal,
+  argTypes: {
+    open: {
+      description: 'Whether the modal is open or not',
+      control: {
+        type: 'boolean',
+      },
+    },
+    loading: {
+      description: 'Whether the settings are loading or submitting',
+      control: {
+        type: 'boolean',
+      },
+    },
+    generatedApiKeyExpiration: {
+      description:
+        'The new generated api key expiration expressed in ISO8601 Timestamp',
+    },
+    generatedApiKey: {
+      description: 'The new generated api key',
+    },
+  },
+  args: {
+    loading: false,
+    open: false,
+    generatedApiKeyExpiration: new Date().toISOString(),
+    generatedApiKey:
+      'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJ0cmVudG9fYXBpX2tleSIsImV4cCI6MTcxMjE1MzE3MCwiaWF0IjoxNzA5NzMzOTcxLCJpc3MiOiJodHRwczovL2dpdGh1Yi5jb20vdHJlbnRvLXByb2plY3Qvd2ViIiwianRpIjoiYmZmMjA0YjUtMzJmMS00YmVlLThiMGItY2IxZGQwNTlmNGRjIiwibmJmIjoxNzA5NzMzOTcxLCJ0eXAiOiJCZWFyZXIifQ.0Lz0MwZaFpIGbSohnkiJ6AN5FFb5Vg5ZVhqM3fdUf3M',
+    onSave: ({ apiKeyExpiration }) =>
```

Comment:
With the previous change, this shouldn't be needed...
PD: but in any case, it should `onGenerated`

Reply:
done with the action

Reply:
Also, onClose should be added to cover all props right?